src/constants.ts

```diff
@@ -38,6 +33,1431 @@ export const EF_JOBS_URL = 'https://ethereum.bamboohr.com/jobs/';
 // ESP external links
 export const ESP_BLOG_URL = 'https://blog.ethereum.org/category/ecosystem-support-program/';
 export const ESP_TWITTER_URL = 'https://twitter.com/EF_ESP';
+export const ESP_EMAIL_ADDRESS = 'mailto:esp@ethereum.org';
+
+// tabs
+export const APPLICANTS_TABS = ['Overview', 'Office Hours', 'Small Grants', 'Project Grants'];
+export const TABS_MAP: TabsMap = {
+  [APPLICANTS_URL]: 0,
+  [OFFICE_HOURS_URL]: 1,
+  [SMALL_GRANTS_URL]: 2,
+  [PROJECT_GRANTS_URL]: 3,
+  [PROJECT_GRANTS_APPLY_URL]: 3,
+  [PROJECT_GRANTS_THANK_YOU_PAGE_URL]: 3
+};
+
+// project categories
+export const PROJECT_CATEGORY_OPTIONS = [
```

Comment:
Since these constants formats are tightly couple with the select component of the form, I would suggest to add it in a new `constants.ts` file under the `/components/forms` folder and keep this constant file shorter and scoped on global aspects.

Reply:
Makes sense 👍